Automattic/woocommerce-payments
# Remove browser based CSV export, default to async server-side export

Fixes #9969
Fixes #9303 by removing JS-based CSV export

#### Changes proposed in this Pull Request

When the number of rows are less than 25, browser based CSV export was used. This is removed. 
Export defaults to async server-side export for transactions, disputes and payouts.

#### Testing instructions
- Test on Disputes, Payouts and Transactions page.
- Ensure less than 25 rows in the page
- Click on the Download button.
- For less than 25 rows, earlier it used to default to client side export with the CSV downloaded immediately. With this branch `update/9764-remove-browser-export`, it will use the server side async export. The export will be generated async and sent by email.

-------------------


**Post merge**
